### DISCUSSION TITLE
v0.3.7

### DISCUSSION THREAD

**Discussion by stefansundin** [stefansundin/privatkopiera#113] [Announcements]
- Fixade problem med urplay.se.
- Fixade problem med nrk.no.

**Full Changelog**: https://github.com/stefansundin/privatkopiera/compare/v0.3.6...v0.3.7

<hr /><em>This discussion was created from the release <a href='https://github.com/stefansundin/privatkopiera/releases/tag/v0.3.7'>v0.3.7</a>.</em>

**Comment by emwtur** [comment]
urplay funkar fint nu 
Tackar !
Donerade en peng till godis

**Comment by Bildo1** [comment]
Det verkar som om alla textfiler är nu tomma tex: ffmpeg -i "https://svt-vod-7a.akamaized.net/d0/world/20220406/5ee1a8de-fa61-4a09-9e25-6f64ce6d0545/text/text-0.vtt" "Varan-tv-stories - Braksketchen.srt" eller: ffmpeg -i "https://svt-vod-4q.akamaized.net/d0/world/20220222/4aa1de58-849c-4186-bf89-9503a3559e54/text/text-0.vtt" "Herr Talman - Avsnitt 2.srt" ( Output file is empty, nothing was encoded (check -ss / -t / -frames parameters if used) ). Vad kan detta berå på?

**Reply by Bildo1** [reply]
Jag märkte att vissa fungerar, tex detta: ffmpeg -i "https://svt-vod-4p.akamaized.net/d0/world/20211228/3a2e976d-202c-4822-87b5-85b751a9ec04/text/text-0.vtt" "Misslyckandena i mitt liv.srt" Konstigt att texten ändå syns på svtplay, även de som kommer laddas tomma med command prompt ...

**Reply by stefansundin** [reply]
Hej. Verkar som att SVT har uppdaterat formatet för undertexterna på nya program och FFmpeg lyckas inte koda av det korrekt. Har skrivit en guide med workaround här: https://github.com/stefansundin/privatkopiera/issues/120

**Reply by Bildo1** [reply]
Fint, tack -skall titta på guiden.

**Comment by Blixtgo** [comment]
Hej Stefan,
Jag får permission denied. Försökt ladda ned från svtplay.se.  ffmp körs allt verkar korrekt, den hittar streamet, sen får jag
Rachmaninovs älskade pianokonsert.mkv: Permission denied

Har försökt med annan fil på svtplay.se . Samma felmedd.

**Reply by stefansundin** [reply]
Du försöker nog skriva till en mapp som din användare inte har behörigheter till (som t.ex. `C:\`). Skriv filerna till din skrivbordsmapp så fungerar det alltid.
